### ISSUE TITLE
nodetool status output should report snapshots file sizes

### ISSUE THREAD

**Issue by lucmargui** [scylladb/scylla#8738]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

The nodetool status output should the full scylla disk utilization (all file sizes) for each node since snapshots are not included/summed to the disk utilization displayed in the output causing a mismatch if compared to the OS disk free verification (df -h) output

tl;dr
scylla disk utilization may differ if comparing the nodetool status output versus the os free disk command df -h since it does not consider snapshots files size
